Patches/.Load2_DesignationCategory_Combined.xml

```diff
@@ -206,6 +206,64 @@
         </match>
     </Operation>
 
+    <!--==========
+    Defenses Expanded
+    ==========-->
+
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Defenses Expanded</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<success>Always</success>
+			<operations>
+				<li Class="PatchOperationReplace">
+					<xpath>/Defs/ThingDef[defName="RazorWireObstacle" or defName="RazorWireBarrier" or defName="Gabion" or defName="HBarrier_Tan" or defName="HBarrier_Green" or defName="HBarrier_FP" or defName="HBarrier_FP_Green" or defName="TBarrier"]/designationCategory</xpath>
+					<value>
+						<designationCategory>Security</designationCategory>
+					</value>
+				</li>
+				<li Class="PatchOperationAdd">
+					<xpath>/Defs/ThingDef[defName="TrapMine_M14AP"]</xpath>
+					<value>
+						<designationCategory>Security</designationCategory>
+					</value>
+				</li>
+				<li Class="PatchOperationRemove">
```

Comment:
Removals need to go into ".Load4_DesignationCategory_Removal.xml"

This is mainly for troubleshooting. In the event that items are missing we can allow old architect tabs to exist and check if items are in them.